README.md

```diff
@@ -41,6 +41,16 @@ Source-build solves common challenges that most developers encounter when trying
 
 Starting with .NET 6, the core source-build infrastructure is integrated into the [dotnet/installer](https://github.com/dotnet/installer/tree/main/src/SourceBuild) repo. The `main` branch on this repo now contains the tooling needed to build .NET's external dependencies from source.
 
+## Linux distributions building and offering .NET
```

Comment:
Missing distros:

CentOS
Homebrew

Reply:
I do not see .NET packages in CentOS and also trying to find the ownership understanding for Homebrew as it is package manager rather than Distro. I will create another PR to include the two.

Reply:
Could we log an issue to track following up on that?

Reply:
Why not resolve this now?

@omajid - can you provide input on how CentOS should be included here?

Regarding Homebrew, yes this one is different because it is a package manager.  In a way, it is similar to the Arch Linux User Repository isn't it?  

What about allowing for multiple feeds per distro as shown below?  I'm not sure on some of the feed labels.  

## .NET in Linux Distribution
| Distro | Package Feed | Maintainer |
|---|---|---|
| Alpine | [Community](https://pkgs.alpinelinux.org/packages?name=dotnet*&branch=v3.16&repo=&arch=&maintainer=)| @ayakael |
| Arch Linux | [Community](https://archlinux.org/packages/?q=dotnet)<br>[Arch User Repo (aur)](https://aur.archlinux.org/packages?K=dotnet) | @alucryd |
| [Fedora](https://fedoraproject.org/wiki/DotNet) | [Default](https://packages.fedoraproject.org/search?query=dotnet) | @omajid, @crummel |
| MacOS | [Homebrew](https://formulae.brew.sh/cask/dotnet) | @asbjornu |
| [Red Hat Enterprise Linux](https://developers.redhat.com/products/dotnet/overview) | [Default](https://access.redhat.com/documentation/en-us/net/6.0)<br>[CentOS Stream Mirror](http://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/) | @omajid |
| [Ubuntu](https://canonical.com/blog/install-dotnet-on-ubuntu) | [Universe](https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=dotnet&searchon=names)<br>[Personal Package Archives (ppa)](https://launchpad.net/ubuntu/+ppas?name_filter=dotnet)| @mirespace |

Reply:
>  @omajid - can you provide input on how CentOS should be included here?

CentOS 7 is the only non-EOL version of CentOS and it doesn't include .NET at all. I guess that leaves us with CentOS Stream. I was thinking of putting CentOS Stream in another row, as a separate distro from RHEL.

Reply:
Updated now